docs/apis/calcite.md

```diff
@@ -0,0 +1,570 @@
+Apache Apex, unified stream processing engine enables application developers to process data at very high throughput with low latency. Though type of data demands different type of
+processing requirements but SQL remains way to process input data .To ensure that existing ETL developers and developers who are well versed with Database applications adopt stream processing application development with ease, integration of SQL with Apex was needed. Being popular Apache project, Apache Calcite was chosen for an integration and to facilitate use of SQL with Apex.
```

Comment:
Being a popular and stable Apache project, Apache Calcite was chose for the integration and to facilitate the use of SQL with Apex.